client/header/index.js

```diff
@@ -80,6 +81,14 @@ class Header extends Component {
 			'is-scrolled': isScrolled,
 		} );
 
+		const notices = [
```

Comment:
Is this testing data?

Reply:
It was test data that was removed but not pushed apparently 😓